docs/sources/tutorials/grafana-fundamentals/index.md

```diff
@@ -165,19 +166,19 @@ Grafana Explore is a workflow for troubleshooting and data exploration. In this
 
 1. Go back to the [sample application](http://localhost:8081) and generate some traffic by adding new links, voting, or just refresh the browser.
 
-1. In the upper-right corner, click the _time picker_, and select **Last 5 minutes**. By zooming in on the last few minutes, it's easier to see when you receive new data.
+1. Back in Grafana, in the upper-right corner, click the _time picker_, and select **Last 5 minutes**. By zooming in on the last few minutes, it's easier to see when you receive new data.
 
 Depending on your use case, you might want to group on other labels. Try grouping by other labels, such as `status_code`, by changing the `by(route)` part of the query.
 
 ## Add a logging data source
 
 Grafana supports log data sources, like [Loki](/oss/loki/). Just like for metrics, you first need to add your data source to Grafana.
 
-1. In the sidebar, hover over the **Configuration** (gear) icon, and then click **Data Sources**.
-1. Click **Add data source**.
+1. In the sidebar, hover over the **Administration** (gear) icon, and then click **Data Sources**.
```

Comment:
```suggestion
1. In the sidebar, click **Connections** and then **Data sources**.
```

Reply:
So just out of curiosity, what's the difference between a data source and a connection? And is there anything different between setting it up using the Admin way, or the Connections way?

Reply:
> So just out of curiosity, what's the difference between a data source and a connection? And is there anything different between setting it up using the Admin way, or the Connections way?

A data source is just a type of connection. In the Cloud version of Grafana, this functionality has been removed from Admin and only exists under Connections. I suspect this will happen in the on-prem version as well, so future-proofing here.